.env.example

```diff
@@ -7,6 +7,7 @@ WAUTH_DB_PORT=
 WAUTH_DB_USER=
 WAUTH_DB_PASS=
 WAUTH_DB_NAME=
+# Use either enabled or disabled
```

Comment:
There's more possible values and `enabled` isn't a valid one (`require` is probably what you want, except that doesn't actually verify the certificate...) - might be better to link out to https://pkg.go.dev/github.com/lib/pq#section-documentation

Reply:
I think I tried with required but it fails, I'll try testing again

Reply:
> Valid values for sslmode are:
> 
> disable - No SSL
> require - Always SSL (skip verification)
> verify-ca - Always SSL (verify that the certificate presented by the server was signed by a trusted CA)
> verify-full - Always SSL (verify that the certification presented by the server was signed by a trusted CA and the server host name matches the one in the certificate)

Reply:
I've just updated it